src/classes/ERR_Handler.cls

```diff
@@ -449,10 +449,10 @@ public class ERR_Handler {
     */
```

Comment:
Since you've added a new field for `Record_Id__c` I definitely recommend updating this class to populate the new field where possible.

Reply:
Is it better/worse to populate this field, given that it is an External Id, albeit, non unique?

Reply:
I think it's better to populate the field. Customers would be able to more easily find errors associated with records. I probably would have found a use for this in my consulting days.